engine/overworld/special_warps.asm

```diff
@@ -2,59 +2,61 @@ SpecialWarpIn::
 	call LoadSpecialWarpData
 	predef LoadTilesetHeader
 	ld hl, wd732
-	bit 2, [hl] ; dungeon warp or fly warp?
+	bit 2, [hl] ; special warp
```

Comment:
Can you please add a `SPECIAL_WARP_F` constant for `2` instead of a comment on this particular usage of it?

Reply:
I would prefer to address that in a separate PR. `wd732` and `wFlags_D733` have multiple bits that should be constants and that alone warrants its own discussion.

Reply:
Regardless, if this line is getting touched at all, it should be with the appropriate fix. No need to add a comment which you know your next PR will remove.

Reply:
Agreed about doing one focused PR with constants for all the bits there. But maybe that should be done first, and then rebase this PR to it.

Reply:
Would it be OK to simply keep the original comments related to this bit for now?

Reply:
Yes.